tox.ini

```diff
@@ -128,11 +128,12 @@ commands_pre =
   protobuf-1: pip install -r {toxinidir}/opentelemetry-proto/test-requirements-1.txt
 
   getting-started: pip install -r {toxinidir}/docs/getting_started/tests/requirements.txt
-  getting-started: pip install -e {env:CONTRIB_REPO}\#egg=opentelemetry-util-http&subdirectory=util/opentelemetry-util-http
-  getting-started: pip install -e {env:CONTRIB_REPO}\#egg=opentelemetry-instrumentation&subdirectory=opentelemetry-instrumentation
-  getting-started: pip install -e {env:CONTRIB_REPO}\#egg=opentelemetry-instrumentation-requests&subdirectory=instrumentation/opentelemetry-instrumentation-requests
-  getting-started: pip install -e {env:CONTRIB_REPO}\#egg=opentelemetry-instrumentation-wsgi&subdirectory=instrumentation/opentelemetry-instrumentation-wsgi
-  getting-started: pip install -e {env:CONTRIB_REPO}\#egg=opentelemetry-instrumentation-flask&subdirectory=instrumentation/opentelemetry-instrumentation-flask
+; See #3879 and https://pip.pypa.io/en/stable/cli/pip/#exists-action-option
```

Comment:
Can you add some more details to #3879? The build logs eventually disappear and it would be nice to have a summary

It's not clear to me which path already exists that it's complaining about or why this issue just popped up

Reply:
Right, I added the logs directly to #3879.

It is also not clear to me why this is happening nor why didn't it happen before.